modules/user/server-ts/shared.ts

```diff
@@ -0,0 +1,36 @@
+import bcrypt from 'bcryptjs';
```

Comment:
It's a bad approach to create one file and put there everything we can. Besides the name tells us nothing about what can it provide.

Reply:
Fixed